File: apiEZPZ/src/main/java/com/example/apiezpz/objectdetection/dto/ObjectDetectionDTO.java
package com.example.apiezpz.objectdetection.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ObjectDetectionDTO {
  private String name;
  private double score;
  private List<Vertex> vertexList;

  @Data
  public static class Vertex{
    private double x;
    private double y;
  }
}

File: backend/src/main/kotlin/com/example/apiezpz/objectdetection/dto/ObjectDetectionDTO.kt
package com.example.apiezpz.objectdetection.dto

data class ObjectDetectionDTO(
    var name: String = "",
    var score: Double = 0.0,
    var vertexList: List<Vertex> = emptyList()
) {
    data class Vertex(
        var x: Double = 0.0,
        var y: Double = 0.0
    )
} 